docs/doxygen/config.doxyfile

```diff
@@ -533,6 +532,13 @@ EXTRACT_LOCAL_METHODS  = NO
 
 EXTRACT_ANON_NSPACES   = NO
 
+# If this flag is set to YES, the name of an unnamed parameter in a declaration
+# will be determined by the corresponding definition. By default unnamed
+# parameters remain unnamed in the output.
+# The default value is: YES.
+
+RESOLVE_UNNAMED_PARAMS = YES
```

Comment:
Should this be set to `NO` instead to discourage using unnamed parameters in function declarations?

Reply:
MISRA already warns us about unnamed parameters